website/package.json

```diff
@@ -9,17 +9,16 @@
     "bootstrap": "^4.6.0",
     "catch-links": "^2.0.1",
     "gatsby": "^2.32.3",
-    "gatsby-image": "^2.11.0",
     "gatsby-plugin-advanced-sitemap": "^1.6.0",
     "gatsby-plugin-catch-links": "^2.10.0",
     "gatsby-plugin-feed": "^2.13.0",
     "gatsby-plugin-google-gtag": "^2.8.0",
     "gatsby-plugin-google-tagmanager": "^2.11.0",
     "gatsby-plugin-react-helmet": "^3.10.0",
     "gatsby-plugin-react-helmet-canonical-urls": "^1.4.0",
-    "gatsby-plugin-robots-txt": "^1.5.5",
+    "gatsby-plugin-robots-txt": "^1.6.2",
     "gatsby-plugin-sass": "^3.2.0",
-    "gatsby-plugin-sharp": "^2.14.1",
+    "gatsby-plugin-sharp": "^2.7.0",
```

Comment:
I wonder if you can remove this too. It has been a while, but IIRC this is imported for use by gatsby-image. 

No harm leaving it in to be safe.

Reply:
@attfarhan it's actually required by the `gatsby-remark-responsive-image`.